.github/workflows/cd-frontend.yml

```diff
@@ -20,6 +20,7 @@ on:
         type: choice
         options:
           - dev
+          - staging
           - prod
 
```

Comment:
...the line you want to change is this one down below

```
environment: ${{ inputs.environment || (github.event_name == 'release' && 'prod') || 'dev' }}
```

change it to (psuedo-code)

```
environment: ${{ inputs.environment || (github.event_name == 'release' && 'prod') || ['dev', 'staging'] }}
```

Reply:
You'll need to do this for the analytics package as well, of course.